packages/client/components/EmailPasswordAuthForm.tsx

```diff
@@ -131,9 +130,10 @@ const EmailPasswordAuthForm = forwardRef((props: Props, ref: any) => {
       const domain = getSSODomainFromEmail(email)
       if (domain && domain !== pendingDomain) {
         setPendingDomain(domain)
+        // Fetch the url to verify SSO is configured for this domain.
+        // Don't cache it as we need a fresh one for login
```

Comment:
Given that the URL is generated on the fly by the server now, I'm curious, since this URL is generated but never used, why can't we use it for login? when does it go invalid?

Reply:
It really depends on the IDP. For CloudFlare it's about a minute. So if they idle a bit between entering their email and pushing the button, it's outdated.

Reply:
a minute, whoa!
we don't send off that request until the email field is blurred, so we can probably get away with caching it, but if the delay isn't too bad it shouldn't matter either way

Reply:
The issue is we cannot really check if it will be rejected because it happens after the redirect, so I would like to stay on the safe side.